force-app/main/default/queues/HOT_Arkiv.queue-meta.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Queue xmlns="http://soap.sforce.com/2006/04/metadata">
+    <doesSendEmailToMembers>false</doesSendEmailToMembers>
```

Comment:
Her kan vi faktisk legge til Arkiv-gruppa. Se eksempel med Akuttvakt-køen i repoet vårt. Beklager for at jeg ikke plukka opp dette ved første review